molecule/redhat/molecule.yml

```diff
@@ -27,6 +32,35 @@ provisioner:
       # space and the slash.  This option converts those characters to
       # underscores when creating the group name.
       force_valid_group_names: always
+  inventory:
+    host_vars:
+      debian9:
```

Comment:
Since this appears to be RHEL/Fedora specific testing, should this remain?

Reply:
Well it does no harm, and it reminds us that this file came from `default/molecule.yml`, so I left it in.

As with the [discussion here](https://github.com/cisagov/ansible-role-xfce/pull/6#discussion_r540648406),one could argue either way.  I think there is a slight benefit to leaving it in.